src/main/java/com/github/dariobalinzo/ElasticSourceConnectorConfig.java

```diff
@@ -116,10 +116,11 @@
     private static final String FIELDS_JSON_CAST_DOC = "Cast to json string instead of parsing nested objects (e.g. order.qty;order.price;status )";
     private static final String FIELDS_JSON_CAST_DISPLAY = "Cast to json string";
 
+    public static final String AVRO_FIELD_CONVERTER = "Avro";
```

Comment:
Here you need to to something like:
```
.define(
                FIELDS_WHITELIST_CONFIG,
                Type.STRING,
                null,
                Importance.MEDIUM,
                FIELDS_WHITELIST_DOC,
                CONNECTOR_GROUP,
                ++orderInGroup,
                Width.MEDIUM,
                FIELDS_WHITELIST_DISPLAY
        )
```

Then you will need to document the new config also in the README. 
Finally the Task class should parse the new config in the start() method, not the costructor.